src/components/App/index.tsx

```diff
@@ -33,10 +25,7 @@ function App() {
           <ModitesContextProvider value={[modites, setModites]}>
             <main role="main">
               <Router>
-                <Route path="/" exact component={Modites} />
-                <Route path="/details" exact component={Details} />
-                <Route path="/details/:id" component={Details} />
-                <Route path="/globe" exact component={Globe} />
+                <Route path="*" component={Modites}/>
```

Comment:
There's no point in Routing if there's just a single catch-all route. You could do just this

```jsx
<main role="main">
  <Modites />
</main>
```